static/includes/apps/SMBLimitations.md

```diff
@@ -0,0 +1,149 @@
+&NewLine;
+
+{{< expand "SMB Limitations" "v" >}}
+
+#### Windows, Linux, and MacOS Client SMB Limitations 
+{{< truetable >}}
+|  | Windows | Linux (Ext4) | Linux (XFS) | MacOS |
```

Comment:
Why are different Linux filesystems listed here? This is SMB client, so the filesystem will be SMB.

Reply:
Each OS client has different limitations. We have users on Windows, Linux and Mac so this is provided for those individuals to know what the limitations are.